config/http/handler/notification.json

```diff
@@ -0,0 +1,31 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^5.0.0/components/context.jsonld",
+  "import": [
+    "css:config/http/handler/handlers/oidc.json",
+    "css:config/app/init/initializers/root.json",
```

Comment:
The server fails when launching with an alternative init configuration, by changing [this import](https://github.com/CommunitySolidServer/CommunitySolidServer/blob/feat/notification-webhooks/config/notification-file.json#L5) to either:

- `"css:config/app/init/initialize-root.json"`
- `"css:config/app/init/initialize-prefilled-root.json"`

This error is printed:

```
Could not create the server
Cause: Detected multiple values for parameter https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^5.0.0/dist/util/handlers/ConditionalHandler.jsonld#ConditionalHandler_source in urn:solid-server:default:RootInitializer. RDF lists should be used for defining multiple values.
```

Here's the fix:

```suggestion
```